src/app/services/evento.service.ts

```diff
@@ -0,0 +1,110 @@
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Evento } from '../models/evento';
+
+const base_url = environment.apiUrl;
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class EventoService {
+  
+
+  constructor(private http: HttpClient) { }
+
+  get token(): string {
+    return localStorage.getItem('token') || '';
+  }
+
+  getEventoPorID(id:string):Promise<Evento>{
+    return new Promise<Evento>(
+      resolve=> {
+        this.http.post(`${base_url}/evento`, {_id:id}).subscribe(data=>{
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138913)
```suggestion
        this.http.post(`${base_url}/evento`, {_id:id}).subscribe((data)=>{
```